src/components/TradeWidget/index.tsx

```diff
@@ -35,6 +36,9 @@ const TradeWidget: React.FC = () => {
   const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId), [fallBackNetworkId])
   const [sellToken, setSellToken] = useState(() => _getToken('DAI', tokens))
   const [receiveToken, setReceiveToken] = useState(() => _getToken('USDC', tokens))
+  const { balances } = useTokenBalances()
+  const sellTokenBalance = useMemo(() => _getToken(sellToken.symbol, balances), [balances, sellToken.symbol])
```

Comment:
You've made another PR with a refactor for `getToken`, I guess we have to write down to adapt this fn too once it's merged

Reply:
Will do :+1: